okhttp-testing-support/src/commonMain/kotlin/okhttp3/TestUtilCommon.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+import okio.FileSystem
+import okio.Path
+import okio.Path.Companion.toPath
+
+val okHttpRoot: Path
```

Comment:
I read your blog on this, it's a shame that this isn't something you can push down to okio testing.

Is there any small hook in gradle we can use to change this? Gradle.propertiea to define that affect launch options?

Reply:
I guess I'd be scanning the "classpath", whatever that means, or other env variables to any way.  From $rootModule, can you find your own test :)

Reply:
I couldn’t find anything in the iOS simulator that traces back to the repo directory.